### DISCUSSION TITLE
Concept Sliders: LoRA Adaptors for Precise Control in Diffusion Models

### DISCUSSION THREAD

**Discussion by realmalikov** [lllyasviel/Fooocus#1030] [Ideas]
Please check out [Concept Sliders: LoRA Adaptors for Precise Control in Diffusion Models](https://sliders.baulab.info/) and implement them in Fooocus. In my opinion, the level of control and quality improvement is excellent. In the section "Sliders to Improve Image Quality" you can see it very well by comparison.

A [tweet](https://twitter.com/RohitGandikota/status/1727410973638852957) from one of the developers with more information.

**Comment by Ambratolm** [comment]
All hail the hands and eyes fix 😃

**Comment by williamsblood** [comment]
wow this looks amazing!

**Comment by jepjoo** [comment]
Tip: Rename the LoRA files from .pt -> .bin and they will work in Fooocus.

**Reply by realmalikov** [reply]
Thank you for the hint.

**Reply by thiagojramos** [reply]
What would these LoRas be? I tried to read the article, but a bunch of calculations appeared, and my mind got tangled up.

**Reply by realmalikov** [reply]
You can download the LoRAs from here: https://sliders.baulab.info/weights/xl_sliders/
repair_slider and fix_hands can be very helpful, and you can also use them at the same time.

**Reply by Ambratolm** [reply]
Is it worth to download everything and try it out ?

**Reply by aimarxjg** [reply]
仅对XL起作用，效果不错
